docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
+This document lists and describes the hardware performance counters and the derived metrics
+available on the AMD Instinct™ MI200 GPU. All hardware performance monitors, and the derived
+performance metrics are accessible via AMD ROCm™ Profiler tool.
 
 ## MI200 performance counters list
 
 ```{note}
-Preliminary validation of all MI200 performance counters is in progress. Those with “[*]” appended to the names require further evaluation.
+Preliminary validation of all MI200 performance counters is in progress. Those with “[*]” appended to
+the names require further evaluation.
 ```
 
 ### GRBM
 
 #### GRBM counters
 
-| Hardware Counter   | Unit   | Definition |
-|--------------------|--------| ------------------------------------------------------|
-| `grbm_count`       | Cycles | Free-running GPU clock |
-| `grbm_gui_active`  | Cycles | GPU active cycles |
-| `grbm_cp_busy`     | Cycles | Any of the command processor (CPC/CPF) blocks are busy. |
-| `grbm_spi_busy`    | Cycles | Any of the shader processor input (SPI) are busy in the shader engine(s). |
-| `grbm_ta_busy`     | Cycles | Any of the texture addressing unit are busy in the shader engine(s). |
-| `grbm_tc_busy`     | Cycles | Any of the texture cache blocks (TCP/TCI/TCA/TCC) are busy. |
-| `grbm_cpc_busy`    | Cycles | The command processor - compute (CPC) is busy. |
-| `grbm_cpf_busy`    | Cycles | The command processor - fetcher (CPF) is busy. |
-| `grbm_utcl2_busy`  | Cycles | The unified translation cache - level 2 (UTCL2) block is busy. |
-| `grbm_ea_busy`     | Cycles | The efficiency arbiter (EA) block is busy. |
+| Hardware counter   | Unit   | Definition |
+| :--------------------| :--------| :-------------------|
+| `GRBM_COUNT` | Cycles | Number of free-running GPU cycles |
+| `GRBM_GUI_ACTIVE` | Cycles | Number of GPU active cycles |
+| `GRBM_CP_BUSY` | Cycles | Number of cycles any of the command processor (CP) blocks are busy |
+| `GRBM_SPI_BUSY` | Cycles | Number of cycles any of the shader processor input (SPI) are busy in the shader engine(s) |
+| `GRBM_TA_BUSY` | Cycles | Number of cycles any of the texture addressing Unit (TA) are busy in the shader engine(s) |
+| `GRBM_TC_BUSY` | Cycles | Number of cycles any of the texture cache (TC) blocks (TCP/TCI/TCA/TCC) are busy |
+| `GRBM_CPC_BUSY` | Cycles | Number of cycles the command processor - compute (CPC) is busy |
+| `GRBM_CPF_BUSY` | Cycles | Number of cycles the command processor - fetcher (CPF) is busy |
+| `GRBM_UTCL2_BUSY` | Cycles | Number of cycles the unified translation cache - level 2 (UTCL2) block is busy |
+| `GRBM_EA_BUSY` | Cycles | Number of cycles the efficiency arbiter (EA) block is busy |
 
 ### Command processor
 
-The command processor counters are further classified into fetcher and compute.
+The command processors (CPs) are further classified into fetcher (CPF) and compute (CPC).
 
 #### CPF
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading (CPF counters).